lib/rules/web/admin_console/4.0/secrets.xyaml

```diff
@@ -99,4 +133,7 @@ check_secret_type:
   element:
     selector:
       xpath: //dt[.='Type']/following::dd[1][.='<secret_type>']
-
+click_edit_secret:
```

Comment:
Seems it's not updated, no need to add `click_edit_secret` IMO, you can just use `click_one_dropdown_action` with `Edit Secret` set as `item`

Reply:
@yapei Updated and passed locally.